server/runtime/src/main/ant/infinispan-server.xml

```diff
@@ -87,30 +87,9 @@
         </sequential>
     </macrodef>
     <target name="-do.kinit" depends="-check.skipped" unless="tests.skipped">
-        <exec executable="bash" outputproperty="ps.pid" resultproperty="ps.result" failifexecutionfails="false"
```

Comment:
Did you mean to include these changes in this PR?

Reply:
No, but the PR with those changes  (#8679) got integrated sometime ago, so I'm not sure why GitHub was still showing them. I pushed again and they disappeared from the diff view.